wpeverest/user-registration
# UR-342 - Remove whitespace before form fields

### All Submissions:






### Changes proposed in this Pull Request:

A whitespace was being displayed before input form fields where field icons are to be displayed in UR pro if enabled. The whitespace has been removed when field icons are not enabled. 

Closes # .

### How to test the changes in this Pull Request:

1. Check if some unnecessary white space is appearing in registration and login forms.

### Types of changes:




### Other information:




### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.
